src/EditorFeatures/Test/Utilities/StackFrameUtils.cs

```diff
@@ -6,6 +6,7 @@
 using System.Collections.Immutable;
 using System.Text;
 using Microsoft.CodeAnalysis.EmbeddedLanguages.StackFrame;
+using Microsoft.CodeAnalysis.EmbeddedLanguages.VirtualChars;
```

Comment:
some things are now extensions.  so that means bringing in the namespace in a few palces.